tests/integration/components/curriculum-inventory-report-overview-test.js

```diff
@@ -67,6 +67,7 @@ module('Integration | Component | curriculum inventory report overview', functio
     await render(hbs`{{curriculum-inventory-report-overview report=report canUpdate=true}}`);
     return settled().then(() => {
       assert.dom('.title').hasText('Overview', 'Component title is visible.');
+      assert.dom('.report-overview-actions .verification-preview').exists({ count: 1 }, 'Rollover course button is visible.');
```

Comment:
Copy paste `'Rollover course button is visible.'` test name.

Reply:
fixed.